testing/scenario_app/android/BUILD.gn

```diff
@@ -0,0 +1,50 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//flutter/testing/scenario_app/runtime_mode.gni")
+
+action("android") {
+  script = "run_gradle.py"
+
+  inputs = [ "$root_out_dir/flutter.jar" ]
+  sources = [
+    "app/build.gradle",
+    "app/src/androidTest/java/dev/flutter/TestRunner.java",
+    "app/src/androidTest/java/dev/flutter/scenarios/EngineLaunchE2ETest.java",
+    "app/src/androidTest/java/dev/flutter/scenarios/ExampleInstrumentedTest.java",
+    "app/src/androidTest/java/dev/flutter/scenariosui/MemoryLeakTests.java",
+    "app/src/androidTest/java/dev/flutter/scenariosui/PlatformTextureUiTests.java",
+    "app/src/androidTest/java/dev/flutter/scenariosui/PlatformViewUiTests.java",
+    "app/src/androidTest/java/dev/flutter/scenariosui/ScreenshotUtil.java",
+    "app/src/androidTest/java/dev/flutter/scenariosui/SpawnEngineTests.java",
+    "app/src/main/AndroidManifest.xml",
+    "app/src/main/java/dev/flutter/scenarios/SpawnedEngineActivity.java",
+    "app/src/main/java/dev/flutter/scenarios/StrictModeFlutterActivity.java",
+    "app/src/main/java/dev/flutter/scenarios/TestActivity.java",
+    "app/src/main/java/dev/flutter/scenarios/TestableFlutterActivity.java",
+    "app/src/main/java/dev/flutter/scenarios/TextPlatformView.java",
+    "app/src/main/java/dev/flutter/scenarios/TextPlatformViewActivity.java",
+    "app/src/main/java/dev/flutter/scenarios/TextPlatformViewFactory.java",
+    "app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java",
+    "build.gradle",
```

Comment:
what do you think about globbing under `app/src`?

Reply:
GN does not like globs.

Reply:
(gradle will still build the files anyway, but GN won't be aware of changes to unmentioned ones dirtying the build)